packages/sail_ui/lib/widgets/inputs/text_field.dart

```diff
@@ -20,6 +20,7 @@ class SailTextField extends StatelessWidget {
   final BoxConstraints? prefixIconConstraints;
   final TextFieldSize size;
   final void Function(String)? onSubmitted;
+  final bool dense;
```

Comment:
@octobocto wyt, it's either

1. opaque modifier, `dense: true`
2. overrides, e.g. `padding: EdgeInsets.zero`
3. new textfield component with the adjusted padding baked in

Reply:
I like `dense: true`!